als-server/shared/src/main/scala/org/mulesoft/als/server/modules/actions/codeactions/CodeActionsHandler.scala

```diff
@@ -0,0 +1,3 @@
+package org.mulesoft.als.server.modules.actions.codeactions
+
```

Comment:
was this added on purpose?

Reply:
Yes, I forgot to delete it and didn't see it in the files.